### PR TITLE
AC-653 : Changed the tab name ((other) to (Tabs and Indents)) in the readme.md…

### PR DESCRIPTION
**Pull request by vish198910**
… file
AC-653 Correcting the README.md 

## Description of what I changed

I have changed the tab name in the README.md in the "Development>Code style" section point number 6.
## Issue I worked on

https://issues.openmrs.org/browse/AC-653

## Checklist: I completed these to help reviewers :)

- [x ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x ] All new and existing **tests passed**.

- [x ] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```
Comment (ribhavsharma) [inline build.gradle]: I don't think that this change is relevant to this PR.

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -1,34 +1,9 @@
...
-          <package name="java" withSubpackages="true" static="false" />
-          <emptyLine />
-          <package name="" withSubpackages="true" static="false" />
-        </value>
-      </option>
-    </JavaCodeStyleSettings>
-    <codeStyleSettings language="JAVA">
-      <option name="RIGHT_MARGIN" value="125" />
-    </codeStyleSettings>
+    <option name="RIGHT_MARGIN" value="125" />
```
Comment (ribhavsharma) [inline .idea/codeStyles/Project.xml]: Same as above

Comment (vish198910) [reply .idea/codeStyles/Project.xml]: Hey, Ribhav it's not about these build Gradle and XML file, My issue was to correct some typos in the README.MD file. That's what I have done.

Comment (ribhavsharma) [reply .idea/codeStyles/Project.xml]: Hi Vishnu, you accidentally made changes to these two files as well, which are not included in the issue desciption. Please see into it :)

Comment (vish198910) [reply .idea/codeStyles/Project.xml]: okAY

Comment (vish198910) [reply .idea/codeStyles/Project.xml]: These changes were not done by they were done while updating my local repository with the master branch.

Comment (ribhavsharma) [reply .idea/codeStyles/Project.xml]: The changes should not appear here if that is so :/

Comment (vish198910) [reply .idea/codeStyles/Project.xml]: But that is indeed the case.

Comment (f4ww4z) [reply .idea/codeStyles/Project.xml]: @vish198910 You shouldn't make changes to this file.

File: build.gradle (hunk 1 above)
Comment (f4ww4z) [reply build.gradle]: @vish198910 did you see this?

> I don't think that this change is relevant to this PR.

**Comment by vish198910** [comment]
Okay I'll correct this soon.

**Comment by vish198910** [comment]
Hey these changes have occurred while  I set up my project in which it was written to update the codestyle

**Comment by vish198910** [comment]
Heyy I am closing this pr and opening a new one for the same issue.

**Comment by vish198910** [comment]
Sorry for the hassle and  delay.

**Comment by ribhavsharma** [comment]
@vish198910 you can always ```git add``` only the files which you want to add in your commit :)

**Comment by vish198910** [comment]
oh yes I know but I was contributing to another issue for the same project but a problem occurred and I was not able to access my repository 
so I have forked again and will be performing the changes soon

**Comment by vish198910** [comment]
btw thanks ribhav for the suggestion

**Comment by vish198910** [comment]
https://github.com/openmrs/openmrs-contrib-android-client/pull/687
This is the new pull request with appropriate changes, I will be careful from the next time.

**Pull request closed without merging**